src/PresentationalComponents/Filters/Filters.js

```diff
@@ -0,0 +1,104 @@
+/* eslint camelcase: 0 */
+import React, { Component } from 'react';
+import { TextInput, Toolbar, ToolbarGroup, ToolbarItem, ToolbarSection } from '@patternfly/react-core';
+import { routerParams } from '@red-hat-insights/insights-frontend-components';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as AppActions from '../../AppActions';
+import PropTypes from 'prop-types';
+import debounce from 'lodash/debounce';
+import DropdownFilters from './DropdownFilters.js';
+
+class Filters extends Component {
+    componentWillUnmount () {
+        this.props.setFilters({ });
+    }
+
+    changeSearchValue = debounce(
+        value => {
+            this.props.setFilters({ ...this.props.filters, text: value });
+            this.props.fetchAction({ ...this.props.filters, text: value });
+        },
+        800
+    );
+
+    addFilter = (param, value) => {
+        const newFilter = this.props.filters[param] ? { [param]: `${this.props.filters[param]}, ${value}` } : { [param]: value };
+        this.props.setFilters({ ...this.props.filters, ...newFilter });
+        this.props.fetchAction({ ...this.props.filters, ...newFilter });
+    };
+
+    removeFilter = (key, value) => {
+        const newFilter = { [key]: this.props.filters[key].split(',').filter(item => item !== value).join(',') };
+
+        if (newFilter[key].length) {
+            this.props.setFilters({ ...this.props.filters, ...newFilter });
+            this.props.fetchAction({ ...this.props.filters, ...newFilter });
+        } else {
+            const filter = Object.assign({}, this.props.filters);
+            delete filter[key];
+            this.props.setFilters(filter);
+            this.props.fetchAction(filter);
+        }
+    };
+
+    render () {
+        const { children, searchPlaceholder, filters, resultsCount, hideCategories } = this.props;
+        return (
+            <Toolbar className='pf-u-justify-content-space-between pf-u-ml-xl pf-u-my-md'>
```

Comment:
Maybe consider using [insights-frontend-components/tableToolbar.md](https://github.com/RedHatInsights/insights-frontend-components/blob/master/doc/components/tableToolbar.md)

Reply:
YEAH! good call, will be following up with another pr to do that.. right now its not a copy and paste fix, @ryelo mentioned it in slack, a the comment https://github.com/RedHatInsights/insights-advisor-frontend/pull/180#issuecomment-463364101 calls it out